src/main/java/seedu/fast/logic/commands/AppointmentCommand.java

```diff
@@ -23,18 +27,36 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Add appointment with the person identified"
             + " by the index number used in the last person listing.\n"
-            + "Parameters: INDEX (must be a positive integer), "
-            + PREFIX_APPOINTMENT + "DATE (must be yyyy-mm-dd) or "
-            + PREFIX_APPOINTMENT + APPOINTMENT_DELETE_COMMAND + "\n"
-            + PREFIX_APPOINTMENT + "[DATE] or " + PREFIX_APPOINTMENT + APPOINTMENT_DELETE_COMMAND + "\n"
-            + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_APPOINTMENT + "2021-10-25 or "
-            + COMMAND_WORD + " 1 "
-            + PREFIX_APPOINTMENT + APPOINTMENT_DELETE_COMMAND;
-
-    public static final String MESSAGE_ADD_APPOINTMENT_SUCCESS = "Added appointment with %1$s: %2$s";
-    public static final String MESSAGE_UPDATE_APPOINTMENT_SUCCESS = "Updated appointment with %1$s: %2$s";
+            + "Parameters (add): INDEX (must be a positive integer), "
+            + PREFIX_APPOINTMENT + "DATE (must be yyyy-mm-dd), "
+            + PREFIX_APPOINTMENT_TIME + "TIME (must be hh:mm (24-hour format)), "
+            + PREFIX_APPOINTMENT_VENUE + "VENUE (recommended to be less than 50 characters long)" + "\n"
```

Comment:
If auto-truncating the string to be 50 chars, should state explicitly that maximum 50 characters is accepted.
Perhaps (has maximum length of 50 characters) would be more clear to the users.